specifications/rsp/settle.yaml

```diff
@@ -70,11 +70,24 @@ components:
           items:
             type: object
             properties:
+              payer_name:
+                type: string
+                description: "Payer name"
+                example: "“Example1 company Pvt. Ltd"
+              payer_address:
+                $ref: 'https://raw.githubusercontent.com/beckn/protocol-specifications/draft/core/v0/api/core.yaml#/components/schemas/Address/properties/full'
               payer_account_no:

```

Comment:
In this section, currency is required since this identifies the account currency. You may want to rename it to something else like Curr_type to keep it unique.